improver_tests/metadata/test_probabilistic.py

```diff
@@ -135,6 +137,30 @@ def test_error_not_probability(self):
             extract_diagnostic_name("lwe_precipitation_rate")
 
 
+class Test_is_probability(IrisTest):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Class name "Test_is_probability" doesn't conform to '[A-Z_][a-zA-Z0-9]+$' pattern](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5435102)